README Updates

Dhruv-J opened the pull request:
This commit updates the README with better steps.

Signed-off-by: Dhruv Jain <2dhruv@gmail.com>
Co-authored-by: Stan Wong <swong394@gmail.com>

antoninbas commented on README.md:
```diff
@@ -20,34 +22,40 @@ determine them. The label can be applied with:
-to `/etc/kubernetes/manifests`. Run the following commands to create the
-directory the repository will be stored on.
+Run the following command to create the directory the repository will be stored
+on:
```
s/stored on/stored in

I think you should specify that this directory should be created on the Node which has been labelled
I wonder if this step is actually necessary, won't the directory be created automatically if needed?

Dhruv-J replied:
At the moment it doesn't, but I'll change the deployment to reflect that.

Dhruv-J replied:
I don't think I can add a configuration to the manifest to create the PV directory if it doesn't exist.

antoninbas replied:
I thought the PV would take care of it by default (or rather kubelet when handling the PV). If not the case, please ignore.

antoninbas replied:
You should still specify that the directory needs to be manually created on the Node that was labelled (i.e. the Node on which the PV will be created).

antoninbas commented on README.md:
```diff
@@ -20,34 +22,40 @@ determine them. The label can be applied with:
-Connect to the control Node and copy `audit-policy.yaml` and `audit-config.yaml`
-to `/etc/kubernetes/manifests`. Run the following commands to create the
-directory the repository will be stored on.
+Run the following command on the node that was just labelled to create the
```
s/the node/the Node

antoninbas reviewed (commented):
one more nit, otherwise LGTM

Dhruv-J committed:
README Updates

This commit updates the README with better steps and adds
a gif of the UI in action.

Signed-off-by: Dhruv Jain <2dhruv@gmail.com>
Co-authored-by: Stan Wong <swong394@gmail.com>
